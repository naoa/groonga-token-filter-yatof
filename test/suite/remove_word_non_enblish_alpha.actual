register token_filters/yatof
[[0,0.0,0.0],true]
tokenize TokenDelimit "EOS こんにちは SEARCH AHWTVXAAGWNDKGG CH2OCH ZH 123AHWUF HOGEHUGA ALALEULYSPROTRPILETHRARGGLNASNPHESERARGCYSLEUGLULEUGLNCYSGLN"   --normalizer NormalizerAuto   --token_filters TokenFilterSkipNonEnglishAlpha
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "eos",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "こんにちは",
      "position": 1,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "search",
      "position": 2,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "ch2och",
      "position": 3,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "zh",
      "position": 4,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "123ahwuf",
      "position": 5,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "hogehuga",
      "position": 6,
      "force_prefix": false,
      "force_prefix_search": false
    }
  ]
]
