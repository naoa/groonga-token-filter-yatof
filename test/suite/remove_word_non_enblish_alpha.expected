register token_filters/yatof

tokenize TokenDelimit "EOS こんにちは SEARCH AHWTVXAAGWNDKGG CH2OCH ZH 123AHWUF HOGEHUGA ALALEULYSPROTRPILETHRARGGLNASNPHESERARGCYSLEUGLULEUGLNCYSGLN" \
  --normalizer NormalizerAuto \
  --token_filters TokenFilterSkipNonEnglishAlpha
